cumulus_etl/common.py

```diff
@@ -153,14 +153,12 @@ def write_json(path: str, data: Any, indent: int = None) -> None:
 
 
 @contextlib.contextmanager
-# pylint: disable-next = contextmanager-generator-missing-cleanup
```

Comment:
Removed these disables - pylint fixed some false positives we were dealing with in their latest release: https://github.com/pylint-dev/pylint/releases/tag/v3.2.2